src/ansys/geometry/core/plotting/plotter_helper.py

```diff
@@ -204,6 +226,16 @@ def compute_edge_object_map(self) -> Dict[pv.Actor, EdgePlot]:
             for edge in object.edges:
                 self._edge_actors_map[edge.actor] = edge
 
+    def enable_picking(self):
+        """Enable picking capabilities in the plotter."""
+        self._pl.scene.enable_mesh_picking(
+            callback=self.picker_callback, use_actor=True, show=False, show_message=False
+        )
+
+    def disable_picking(self):
```

Comment:
It seems that enabling picking is activated by default... how do you disable it then?

Reply:
I can see the method, but no actual call to it.

Reply:
It is called when activating the measurement widget, so it doesn't interfere with each other. 

The function is called here: https://github.com/ansys/pyansys-geometry/blob/c7b52122e06b512c58ed727a13e38e3ca202a0bf/src/ansys/geometry/core/plotting/widgets/measure.py#L71-L74

Regarding the first comment, I forgot to use the `allow_picking` flag to the proper places, already fixed 🙂